spec/models/searchgov_url_spec.rb

```diff
@@ -381,6 +381,42 @@
       end
     end
 
+    context 'when the searchgov domain has js renderer enabled' do
+      before do
+        allow(searchgov_url).to receive(:searchgov_domain).and_return(domain_with_js_renderer)
+        allow(JsFetcher).to receive(:fetch)
+      end
+
+      # Below live site will pinged on each test run, should be handled with VCR cassette config.
```

Comment:
This comment can be removed, as these specs aren't actually pinging a live url.  (If you turned off your wifi and ran this locally, it'd still pass because of the vcr_cassettes added below.)